types/src/front/lib/assistant.ts

```diff
@@ -341,9 +344,27 @@ export const CLAUDE_3_OPUS_DEFAULT_MODEL_CONFIG: ModelConfigurationType = {
   supportsVision: true,
   toolUseMetaPrompt: ANTHROPIC_TOOL_USE_META_PROMPT,
 };
+
+export const CLAUDE_3_5_SONNET_20240620_DEPRECATED_MODEL_CONFIG: ModelConfigurationType =
+  {
+    providerId: "anthropic",
+    modelId: CLAUDE_3_5_SONNET_20240620_MODEL_ID,
+    displayName: "Claude 3.5 Sonnet",
+    contextSize: 180_000,
+    recommendedTopK: 32,
+    recommendedExhaustiveTopK: 128, // 65_536
+    largeModel: true,
+    description: "Anthropic's latest Claude 3.5 Sonnet model (200k context).",
```

Comment:
Maybe a small distinction in the `description` so users can see the differences?

Reply:
This is not visible and will be removed

Reply:
I think the goal is that they don't?

Reply:
They literally just don't see this I think

Reply:
And if in an unfortunate turn of events they did manage to see it I wouldn't want them to see something weird indeed